### PR TITLE
media-fonts/iosevka: Version bump to 17.0.2

### PR DESCRIPTION
**Pull request by listout**
Signed-off-by: brahmajit das <listout@protonmail.com>

### CODE REVIEW

File: media-fonts/iosevka/iosevka-17.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+HOMEPAGE="https://be5invis.github.io/Iosevka/"
+SRC_URI="https://github.com/be5invis/Iosevka/releases/download/v${PV}/super-ttc-${P}.zip"
+
+LICENSE="OFL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~x86"
+
+BDEPEND="app-arch/unzip"
+
+S="${WORKDIR}"
```
Comment (ceamac) [inline media-fonts/iosevka/iosevka-17.0.2.ebuild:18]: Please move `S` below `SRC_URI`

**Review by ceamac — CHANGES_REQUESTED** [review]
Please update the HOMEPAGE
`RedirectedUrl: version 17.0.2: HOMEPAGE: permanently redirected: https://be5invis.github.io/Iosevka/ -> https://typeof.net/Iosevka/`

File: media-fonts/iosevka/iosevka-17.0.2.ebuild (hunk 1 above)
Comment (listout) [reply media-fonts/iosevka/iosevka-17.0.2.ebuild]: done

**Review by juippis — COMMENTED** [review]
I see you want to become a maintainer, could you look into https://bugs.gentoo.org/769263 and comment whether it's even relevant anymore? I see the font version has taken a huge leap from the days that bug was created.

**Pull request closed without merging**